app/assets/stylesheets/_item.scss

```diff
@@ -0,0 +1,31 @@
+#item-title {
+  font-weight: bold;
+  margin-top: 20px;
+  margin-left: 30px;
+  text-shadow: 0px 2px 3px #666;
```

Comment:
Color literals like `#666` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>`0px` should be written without units as `0`